### PR TITLE
Added 'Default Media Type' setting to the 'Import Swagger' and 'New Project' dialogs

### PR DESCRIPTION
**Pull request by VjacheslavVytjagov**
Added 'Default Media Type' setting and its using in the Swagger2Importer

### CODE REVIEW

File: src/main/groovy/com/smartbear/swagger/SwaggerUtils.groovy [hunk 1]
```diff
@@ -21,13 +22,14 @@ class SwaggerUtils {
      *
      * @param url
      * @param project
+     * @param defaulMediaType
      * @return the corresponding SwaggerImporter based on the described "algorithm"
      */
 
-    static SwaggerImporter createSwaggerImporter(String url, WsdlProject project) {
+    static SwaggerImporter createSwaggerImporter(String url, WsdlProject project, String defaulMediaType) {
```
Comment (olensmar) [inline src/main/groovy/com/smartbear/swagger/SwaggerUtils.groovy]: please keep the old constructor that passes on to the new one with the default value from SwaggerUtils - I know we have others that use these classes and I don't want their code to break.

File: src/main/groovy/com/smartbear/swagger/Swagger2Importer.groovy [hunk 2]
```diff
@@ -47,10 +47,16 @@ import org.slf4j.LoggerFactory
 class Swagger2Importer implements SwaggerImporter {
 
     private final WsdlProject project
+    private final String defaultMediaType;
     private static Logger logger = LoggerFactory.getLogger(Swagger2Importer)
 
-    public Swagger2Importer(WsdlProject project) {
+    public Swagger2Importer(WsdlProject project, String defaultMediaType) {
```
Comment (olensmar) [inline src/main/groovy/com/smartbear/swagger/Swagger2Importer.groovy]: see comment on constructor below

**Comment by VjacheslavVytjagov** [comment]
You just did not notice...
I kept old versions in the both cases like this:

    public Swagger2Importer(WsdlProject project, String defaultMediaType) {
        this.project = project
        this.defaultMediaType = defaultMediaType
    }

    public Swagger2Importer(WsdlProject project) {
        this(project, "application/json")
    }

and

    static SwaggerImporter createSwaggerImporter(String url, WsdlProject project) {
        return createSwaggerImporter(url, project, DEFAULT_MEDIA_TYPE)
    }

**Comment by olensmar** [comment]
thanks :-)

**Pull request merged**